Platform/NXP/Sabre_iMX6SX_1GB/Drivers/GraphicsOutputDxe/Lcdif.c

```diff
@@ -0,0 +1,220 @@
+    DebugPrint(DEBUG_INFO, "CSC_OFFSET         %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_CSC_OFFSET   ));   
+    DebugPrint(DEBUG_INFO, "CSC_LIMIT          %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_CSC_LIMIT    ));  
+    DebugPrint(DEBUG_INFO, "DATA               %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DATA         ));
+    DebugPrint(DEBUG_INFO, "BM_ERROR_STAT      %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_BM_ERROR_STAT));      
+    DebugPrint(DEBUG_INFO, "CRC_STAT           %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_CRC_STAT     )); 
+    DebugPrint(DEBUG_INFO, "STAT               %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_STAT         ));
+    DebugPrint(DEBUG_INFO, "VERSION            %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_VERSION      ));
+    DebugPrint(DEBUG_INFO, "DEBUG0             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DEBUG0       ));
+    DebugPrint(DEBUG_INFO, "DEBUG1             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DEBUG1       ));
+    DebugPrint(DEBUG_INFO, "DEBUG2             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DEBUG2       ));
+    DebugPrint(DEBUG_INFO, "THRES              %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_THRES        ));
+    DebugPrint(DEBUG_INFO, "AS_CTRL            %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_AS_CTRL      ));
+    DebugPrint(DEBUG_INFO, "AS_BUF             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_AS_BUF       ));
+    DebugPrint(DEBUG_INFO, "AS_NEXT_BUF        %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_AS_NEXT_BUF  ));    
+    DebugPrint(DEBUG_INFO, "AS_CLRKEYLOW       %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_AS_CLRKEYLOW ));     
+    DebugPrint(DEBUG_INFO, "AS_CLRKEYHIGH      %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_AS_CLRKEYHIGH));      
+    DebugPrint(DEBUG_INFO, "SYNC_DELAY         %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_SYNC_DELAY   ));   
+    DebugPrint(DEBUG_INFO, "DEBUG3             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DEBUG3       ));
+    DebugPrint(DEBUG_INFO, "DEBUG4             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DEBUG4       ));
+    DebugPrint(DEBUG_INFO, "DEBUG5             %08X\n", MmioRead32(IMX6SX_PHYSADDR_LCDIF1_DEBUG5       ));
+#endif
+}
+
+VOID
+InitLCD(
+    VOID
+    )
+{
+    DEBUG((DEBUG_INFO,"InitLCD\n"));
+
+    // turn on LCD clocks
+    ImxClkPwrLcdClockEnable(IMX6SX_PHYSADDR_LCDIF2);
+
+    // set run bit
+    MmioWrite32(IMX6SX_PHYSADDR_LCDIF2_RL_SET, IMX6SX_LCDIF_RL_RUN);
+
+    return;
+}
+
+EFI_STATUS
+ConfigureLCD(
+    IMX_DISPLAY_TIMING* Timing,
+    VOID* FrameBuffer
+    )
+{
+    EFI_STATUS status;
+
+    DEBUG((DEBUG_INFO,"ConfigureLCD\n"));
+    status = ImxSetLcdIfClockRate(IMX6SX_PHYSADDR_LCDIF2, Timing->PixelClock);
+    if (EFI_ERROR(status)) {
+        DebugPrint(DEBUG_ERROR, "ConfigureLCD - ImxClkPwrLcdIfEnable failed");
+        return status;
+    }
+
+    sResetLCDIF();
+
+    // Debug code to allocate as much bandwidth as possible to LCDIF
+#if 0
```

Comment:
No `#if 0` code. Wrap it with an LCDIF_DEBUG define check.

Reply:
Fixed.